doc/dev_manual/image_viewer.rst

```diff
@@ -0,0 +1,501 @@
+API examples we assume the viewer object is contained in variable `v`.
+
+Loading the Viewer
+==================
+
+The image viewer can load data in a number of formats, but all formats
+are wrapped with a class that corresponds to the *model* part of the
+model-view-controller design used by Ginga.
+For more information on creating the wrapper object,
+see :ref:`ch-image-data-wrappers`.
+
+Once you have successfully loaded an image wrapper, you can set it into
+the image viewer::
+
+  >>> v.set_image(img)
+
+
+======================================
+Manipulating the view, programatically
+======================================
+
+The sections below describe how to manipulate the view programatically.
+Many of the API calls simply set a value in the viewer's settings
+object; if so, we note the associated keyword for that setting.
+
+
+Scaling
+=======
+
+In Ginge lexicon, "scaling" is setting a precise scale factor for pixels.
+For example a scale factor of 2.0 means that two pixels are shown in the
+viewer for every one pixel of data.
+
+Ginga allows different scale factors for the X and Y axes.
+
+Set a precise scale for X and Y::
+
+  >>> v.set_scale((0.5, 0.5))
+
+.. note: Corresponds to the viewer setting `scale`.
+
+Get the scale for X and Y::
+
+  >>> v.get_scale_xy()
+  (0.5, 0.5)
+
+If the scale factors are always set to the same value and adjusted
+equally in tandem, then you can obtain the general scale value as::
+
+  >>> v.get_scale_max()
+  0.5
+
+  
+Scaling Limits
+--------------
+
+Ginga places a limit on the scale when the scale would result in an
+abberant viewing condition.  For example, it will cease to increase the
+scale when the pixel size approaches the window size.
+
+Ginga has a feature for setting arbitrary hard limits on the minimum and
+maximum scale.
+
+To get the scale limits (returns minimum and maximum scale)::
+
+  >>> v.get_scale_limits()
+  (1e-05, 10000.0)
+
+To set the scale limits::
+
+  >>> v.set_scale_limits(min_scale, max_scale)
+
+The limits apply to all dimensions.  A value of `None` (the default)
+means that no artificial limit will be imposed.
+
+
+Zooming
+-------
+
+Although Ginga uses scale factors to accomplish image scaling, it also
+supports the concept of zooming via *levels*.  It does this by mapping a
+zoom level to a scale factor according to the *zoom algorithm* (see
+following section) in use. 
+
+Zoom algorithms assume that a 1:1 scale is always zoom level 0.
+Providing a negative zoom level zooms "out" (smaller scale) while a
+positive value zooms "in" (larger scale).  
+
+Set a zoom level::
+
+  >>> v.zoom_to(4)
+
+Zoom in one level::
+
+  >>> v.zoom_in()
+
+Zoom out three levels::
+
+  >>> v.zoom_out(3)
+
+Zoom so that the loaded image fits the window::
+
+  >>> v.zoom_fit()
+
+Get the current zoom level corresponding to the current scale::
+
+  >>> v.get_zoom()
+  -8.544
+
+.. note: Zoom levels need not be integers, and there should be an
```

Comment:
`.. note::` (double colon)

And I now notice that this is being repeated in many places; if this is a mistake, please correct them all.